scripts/README.md

```diff
@@ -0,0 +1,64 @@
+---
+page_type: script
+languages:
+- powershell
+products:
+- azure
+- azure-digital-twins
+description: "How to deploy Azure Digital twins using automated powershell scripts"
+---
+
+# Use Automated scripts to deploy Azure Digital Twins
+Deploy.ps1 is an Azure Digital Twins code sample used to deploy an Azure Digital Twins instance and set the required permissions. It can also be used as a starting point for writing your own scripted interactions.
+
+## Prerequisites
+
+* You'll need Azure subscription. You can set one up for free [here](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* You'll need a resource group in your Azure subscription. You can create your resource group using [this](https://docs.microsoft.com/azure/azure-resource-manager/management/manage-resource-groups-cli) article.
```

Comment:
You can set up a resource group as part of the script flow if you enter the name of a new rg when it asks. So this one is not a required prereq